### DISCUSSION TITLE
external_bar and native-fullscreen

### DISCUSSION THREAD

**Discussion by blink-twice** [asmvik/yabai#930] [Q&A] [answered]
Is the external_bar option supposed work on spaces that have windows in native-fullscreen? The wiki does not have any indication that external_bar applies **only** to spaces in which windows are either **not fullscreen** or are **in zoom-fullscreen** mode. 

This is what I'm trying to achieve - for my most frequently used apps, I use one space per app with each window in native-fullscreen mode. I want simple-bar to be visible at the bottom of each space, but currently it looks like this is not possible unless I use the windows in zoom-fullscreen. Is this correct? I am using v.3.3.8

**Comment by asmvik** [comment, marked as answer]
Yabai does not interfere with macOS Fullscreen spaces in any way. It is not possible to control the size of applications/windows that run in fullscreen space, and so it is impossible to create padding etc. Most windows should not be able to overlap/become visible in a fullscreen space anyway, although technically it is possible to make that happen.
